eZ/Bundle/EzPublishCoreBundle/Resources/config/fieldtype_services.yml

```diff
@@ -12,6 +12,256 @@ parameters:
     ezpublish.fieldType.ezpage.hashConverter.class: eZ\Publish\Core\FieldType\Page\HashConverter
 
     # RichText
+    ezpublish.fieldType.ezrichtext.normalizer.character_entity_definition.class: eZ\Publish\Core\FieldType\RichText\Normalizer\CharacterEntityDefinition
+    ezpublish.fieldType.ezrichtext.normalizer.html_entities.data:
+        nbsp: 160
```

Comment:
I agree with glye, why don't you write that in a DTD file instead of auto-generating it for that list ?